// tsparticles-preset-triangles/umd/options.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    exports.options = void 0;
    exports.options = {
        background: {
            color: "#000000",
        },
        particles: {
            number: {
                value: 100,
            },
            links: {
                distance: 125,
                enable: true,
                triangles: {
                    enable: true,
                    opacity: 0.1,
                },
            },
            move: {
                enable: true,
                speed: 5,
            },
            size: {
                value: 1,
            },
            shape: {
                type: "circle",
            },
        },
    };
});
